e2e/cypress/integration/picking/pick_HUs_and_create_shipment.js

```diff
@@ -91,6 +91,8 @@ describe('Pick the SO', function() {
     cy.executeQuickActionWithRightSideTable('WEBUI_Picking_HUEditor_Launcher');
 
     cy.selectItemUsingBarcodeFilter(huValue1);
```

Comment:
`selectItemUsingBarcodeFilter` should be added to cypress/support/index.d.ts, with documentation. this helps IDEs figure out how to use the function.